.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,577 @@
+					button-default:
+						(
+							color: var(--global-neutral-10),
+							background-color: var(--global-white),
+						),
+					button-default-hover: (
+						background-color: var(--global-neutral-1),
+					),
+					// DESIGN ASSUMPTION: Date does not require component styling.
+					// DESIGN ASSUMPTION: Grid does not require component styling.
+					carousel:
+						(
+							--slides: 1,
+						),
+					carousel-actions: (
+						display: none,
+						padding-left: var(--global-spacing-3),
+						padding-right: var(--global-spacing-3),
+						//
+						// The width should place the action buttons near the edge of the carousel responsively
+						//
+						// width is derived from adding
+						//   --layout-content-max-width
+						//   2x button widths (--global-font-size-9)
+						//   2x button margins (--global-font-size-5)
+						// then dividing those by
+						//   --layout-max-width
+						// using the formula: (48 + 32 + 1216 + 32 + 48) / 1440
+						//
+						width:
+							calc(calc(var(--layout-content-max-width) + 160) / var(--layout-max-width) * 100%),
+					),
+					carousel-button: (
+						box-shadow: var(--global-box-shadow-1),
+						border-radius: var(--global-spacing-1),
+						background-color: var(--global-white),
+						height: var(--global-spacing-7),
+						width: var(--global-spacing-7),
+					),
+					carousel-track: (
+						gap: var(--global-spacing-2),
+						width: var(--layout-content-scale-width),
+					),
+					heading: (
+						font-weight: var(--global-font-weight-7),
+					),
+					icon: (
+						fill: var(--global-neutral-8),
+						height: var(--global-spacing-4),
+						width: var(--global-spacing-4),
+					),
+					input: (
+						align-items: flex-start,
+						display: flex,
+						flex-direction: column,
+						gap: var(--global-spacing-1),
+					),
+					// DESIGN QUESTION:
+					// There are multiple paddings for input in Figma.
+					// Is this the correct one for a standard, default text input?
+					input-input:
+						(
+							border: 1px solid var(--global-neutral-5),
+							border-radius: var(--global-border-radius-1),
+							padding: var(--global-spacing-1) var(--global-spacing-2),
+						),
+					// DESIGN ASSUMPTION:
+					// This focused input outline is a close but not exact match to the
+					// design in Figma. In figma, the outline inner border is 1px with
+					// outer border of 2px. This 3px double outline results in a 2px
+					// inner border with 1px outer border. Was there another
+					input-input-focus:
+						(
+							outline: 3px double var(--global-neutral-5),
+						),
+					input-label: (
+						display: block,
+						font-weight: var(--global-font-weight-7),
+					),
+					// DESIGN ASSUMPTION:
+					// The border color in Figma is not one of the red
+					// color tokens. I took my best guess with global-red-4.
+					input-error-input:
+						(
+							border-color: var(--global-red-4),
```

Comment:
looks like the border color is alias token form danger #B5646

Yeah I couldn't find that in the reds either. red-4 is porbably the closest with b5... 

![Screen Shot 2022-04-27 at 14 26 04](https://user-images.githubusercontent.com/5950956/165604468-2699bc72-3f42-4989-a2d3-c1ba4944cf86.png)

Reply:
Yes. I believe @brannytempy and @haganka are working on this.
I put in notes for best estimates in the meantime.

Reply:
Updated with global color values provided by Design.